kfs-core/src/main/java/edu/arizona/kfs/sys/businessobject/RouteCode.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2009 The Kuali Foundation.
+ * 
+ * Licensed under the Educational Community License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.opensource.org/licenses/ecl1.php
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.arizona.kfs.sys.businessobject;
+
+import java.util.LinkedHashMap;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+
+
+/**
+ * @author Kevin McOmber
+ * 
+ */
+public class RouteCode extends PersistableBusinessObjectBase {
```

Comment:
As this BO has an active indicator, should also implement MutableInactivatable. See BO class TaxRegion for an example.